plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -390,29 +390,48 @@ export class GraphQLInstrumentation extends InstrumentationBase {
     });
   }
 
+  private getOperationType(
+    operation: graphqlTypes.DefinitionNode | undefined
+  ): string | undefined {
+    if (operation && operation.kind == 'OperationDefinition') {
+      return operation.operation;
+    }
+
+    return undefined;
+  }
+
+  private getOperationName(
+    operation: graphqlTypes.DefinitionNode | undefined,
+    processedArgs: graphqlTypes.ExecutionArgs
+  ): string | undefined {
+    if (operation && operation.kind == 'OperationDefinition') {
```

Comment:
please use es6 and strict comparison
```suggestion
    if operation?.kind === 'OperationDefinition') {
```